nixos/modules/services/misc/autorandr.nix

```diff
@@ -22,19 +22,9 @@ in {
 
     environment.systemPackages = [ pkgs.autorandr ];
 
-    # systemd.unitPackages = [ pkgs.autorandr ];
+    systemd.packages = [ pkgs.autorandr ];
```

Comment:
I have enabled the service and it also seems to be trigger:

```
services.autorandr.enable = true;
```

```
Jun 08 12:08:56 turingmachine autorandr[12586]: Failed to load profile 'default': Profile not found (line 1035)
Jun 08 12:08:56 turingmachine autorandr[12586]: Running autorandr as joerg for display :0
Jun 08 12:08:56 turingmachine systemd[1]: Started autorandr execution hook.
```

So the idea would be to make a `default` profile in my user account for autorandr?